src/communication/network_interface_sr.c

```diff
@@ -10849,13 +10849,17 @@ sflashback_get_summary (THREAD_ENTRY * thread_p, unsigned int rid, char *request
       goto error;
     }
 
+  FLASHBACK_SET_REQUEST_TIME ();
+
   /* get summary list */
   error_code = flashback_make_summary_list (thread_p, &context);
   if (error_code != NO_ERROR)
     {
       goto error;
     }
 
+  FLASHBACK_SET_MIN_LOG_PAGEID_TO_KEEP (&context.start_lsa);
```

Comment:
What if the log volume is truncated between `flashback_make_summary_list()` and here? It seems that logs after `context.start_lsa` are possibly deleted.

Reply:
Yes, you are right. 
I will set the flag that represents flashback state (run or stop), and prevent the archive log volumes from being deleted.